Source/Core/DiscIO/NANDContentLoader.cpp

```diff
@@ -398,16 +398,16 @@ void CNANDContentLoader::RemoveTitle() const
   }
 }
 
-cUIDsys::cUIDsys()
+cUIDsys::cUIDsys(Common::FromWhichRoot root)
 {
-  UpdateLocation();
+  Update(root);
 }
 
-void cUIDsys::UpdateLocation()
+void cUIDsys::Update(Common::FromWhichRoot root)
```

Comment:
I can't think of any case where we couldn't simply make a new object… I'll move Update() to the constructor.